ui-admin/src/forms/designer/SplitCalculatedValueDesigner.tsx

```diff
@@ -0,0 +1,67 @@
+import { FormContent } from '@juniper/ui-core'
+import React from 'react'
+import { Button } from 'components/forms/Button'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { CalculatedValue } from 'survey-core'
+import { SplitCalculatedValueEditor } from 'forms/designer/SplitCalculatedValueEditor'
+import {
+  DocsKey,
+  ZendeskLink
+} from 'util/zendeskUtils'
+
+/**
+ * A split-view designer for calculated values. On the left side, you can edit the calculated value expression,
+ * and on the right side you can see the questions that are used in the expression. If you fill out the questions,
+ * you can get a preview of how the value will be calculated.
+ */
+export const SplitCalculatedValueDesigner = ({ content, onChange }: {
+  content: FormContent, onChange: (newContent: FormContent) => void
+}) => {
+  return <>
+    <span className="ms-3 mt-3">
+      Derived values allow you to create hidden data calculated based on survey responses.
+      See <ZendeskLink doc={DocsKey.DERIVED_VALUES}>here</ZendeskLink> for more information.
+    </span>
+    <div className="container w-100">
+      <div className="d-flex">
+        {renderNewElementButton(content, onChange, -1)}
+      </div>
+      {content && content.calculatedValues && content.calculatedValues.map((_, elementIndex) => (
+        <div key={elementIndex} className="container">
+          <SplitCalculatedValueEditor
+            calculatedValueIndex={elementIndex}
+            editedContent={content}
+            onChange={onChange}/>
+          <div className="d-flex">
+            {renderNewElementButton(content, onChange, elementIndex)}
+          </div>
+        </div>
+      ))}
+    </div>
+  </>
+}
+
+const renderNewElementButton = (
+  formContent: FormContent,
+  onChange: (newContent: FormContent) => void,
+  calculatedValueIdx: number) => {
+  return <div className="my-2">
```

Comment:
will be clearer if this function just takes responsibility for rendering the button, not a surrounding div.  You can add "py-2" to the classes of the surrounding div above to get the effect you need